jekyll/_cci2/sample-config.md

```diff
@@ -78,4 +83,145 @@ workflows:
               only: master
 ```
 {% endraw %}
-This example shows a sequential workflow with the `test` job configured to run only on the master branch. Refer to the [Workflows]({{ site.baseurl }}/2.0/workflows) document for complete details about orchestrating job runs with parallel, sequential, fan-in, and fan-out workflows.
+This example shows a sequential workflow with the `test` job configured to run only on the master branch. Refer to the [Workflows]({{ site.baseurl }}/2.0/workflows) document for complete details about orchestrating job runs with parallel, sequential, and manual approval workflows.
+
+## Sample Configuration with Fan-in/Fan-out Workflow
+Following is a sample configuration for a Fan-in/Fan-out workflow. Refer to [the complete demo repo on GitHub](https://github.com/CircleCI-Public/circleci-demo-workflows/blob/fan-in-fan-out/.circleci/config.yml) for details.
+
+{% raw %}
+```
```

Comment:
Well, I'd like to try having this example in the doc to see what people say and if it helps people feel they have enough real examples in the docs to work from.